packages/runtime-core/src/component.ts

```diff
@@ -273,7 +273,7 @@ export function setupStatefulComponent(
 
 export function handleSetupResult(
   instance: ComponentInternalInstance,
-  setupResult: unknown,
```

Comment:
What's the point of these `unknown` to `any` changes? The result should be unknown here.